tests/platform_tests/api/platform_api_test_base.py

```diff
@@ -0,0 +1,32 @@
+from common.helpers.assertions import pytest_assert
+
+
+class PlatformApiTestBase(object):
+    """Platform API test base class"""
+
+    failed_expectations = []
+
+    def expect(self, expr, err_msg):
```

Comment:
make this a shared helper?

Reply:
I considered it. We might be able to move it to common/ in the future?